src/arvmiscprivate.h

```diff
@@ -33,4 +35,13 @@ struct _ArvValue {
 	} data;
 };
 
+typedef struct _ArvIfaceAddr ArvIfaceAddr;
+
+struct _ArvIfaceAddr{
+	struct sockaddr *addr, *netmask, *broadaddr;
```

Comment:
One line per member definition please.
Don't make this structure public, it should go to arvnetwork.c.